packages/antd/src/widgets/SelectWidget/index.js

```diff
@@ -64,8 +64,6 @@ const SelectWidget = ({
 
   const { enumOptions, enumDisabled } = options;
 
-  const emptyValue = multiple ? [] : '';
```

Comment:
Why remove this? I think this is required for multiple select support?

Reply:
I thought so too, but seems like Ant Design component handles it just fine. Having the `emptyValue` however breaks the placeholder, so that's a bug there, and hence why I removed it. You can try it in the playground.